EDU-709 - splitting export page (#3042)
temporalio/documentation: docs/production-deployment/cloud/export.mdx

:::tip Support, stability, and dependency info

- Workflow History Export is in a Public Preview release status for Temporal Cloud.

> - Workflow History Export does not currently support Namespaces in Google Cloud.

Reviewer: Is the intent to have the whole GCP documentation ready, but still add a mention it's not supported here?
Or should we have the documentation for GCP ready, but just merge the part saying it's not supported for now?

Wondering because right now it could be confusing if I get directly to the part below saying `you can configure your workflow history export in AWS and GCP currently`.

Reviewer: Also wondering if the `this is not supported in GCP` should be in a warning block vs. a tip block; tip blocks are easier to miss in docusaurus

Author: @fairlydurable What do you recommend here?

Reviewer 2: We do not use "currently" as that is a promise of future changes in direction. Docs only reflects items in our release stages

Author: Sounds good.

>    - **Schedule**: The Export job is scheduled to run on an hourly basis, starting at 10 minutes past each hour.

Reviewer 3: ```suggestion
The Export job runs as an hourly process, starting at 10 minutes past each hour.
```

for active voice

   - **Duration**: The time taken for the export process can vary, so it might not be instantaneous.

2. **Checking the Object Storage**:

   - **Files Arrival**: Post the initial hour of setting up, inspect your object storage.
     You should see the exported Workflow History files.
     These files encapsulate the relevant data from the Closed Workflows.
   - **Directory Structure**: Your exported files will adhere to the following naming convention and path:

>    ```bash

Reviewer: This is not bash though, maybe just don't specify any language?

Author: Fixed
